core/store/src/lib.rs

```diff
@@ -330,11 +335,19 @@ pub fn get_code(
         .map(|opt| opt.map(|code| ContractCode::new(code.to_vec())))
 }
 
+pub fn get_account_id_by_hash(
+    state_update: &TrieUpdate,
+    account_id_hash: CryptoHash,
+) -> Result<Option<AccountId>, StorageError> {
+    get(state_update, KeyForAccountId::from_hash(account_id_hash))
```

Comment:
How do we handle hash collision?

Reply:
It's sha256

Reply:
I still feel uneasy relying on the fact that there are no known collisions so far, but ok.

Reply:
Do we really need a hash function with strong cryptography properties for the trie key? I see it being cost-ineffective, and we also implicitly rely on a fact that there are no collisions (which is not true, there is just a very small chance), but that can lead us into a corrupted state when two accounts collide by the hash.

/cc @nearmax

Reply:
I think we're safe from the perspective of `account IDs` requiring a certain format. Even if a hash collision being discovered in the future, the chance that it fits `account ID` format is even lower.
We'll be able to upgrade state if suddenly there are sha256 collision being disovered.